### PR TITLE
CHORE: Further PULL_REQUEST_TEMPLATE tweaks

### PR DESCRIPTION
**Pull request by ababic**
## About these changes

* Fix weird code-block formatting
* Simplify structure and language
* Move common scenario (merging a feature branch into dev) above the release scenario
* Remove the 'check that merging to develop is okay' line - this should always be okay for approved changes
* Be more specific about where to check / communicate about deployments

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -17,24 +16,19 @@ Related ticket(s):
 - [ ] Waited for all CI jobs to pass before requesting a review.
 - [ ] Added/updated tests and documentation where relevant.
 
-## For Reviewer/Developer
+## Merging a feature branch?
```
Comment (TNA-Allan) [inline .github/PULL_REQUEST_TEMPLATE.md:20]: The branch in heading can also be `chore/` `fix/`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -17,24 +16,19 @@ Related ticket(s):
...
-        - select `Squash and merge`
-
-- After merging PR:
-
-  - check and arrange to deploy on platform-sh.
-  - update status on JIRA ticket on deployment.
-  - update slack on deployment progress of the JIRA ticket once deployed.
+1. Check on the `#dev-etna` Slack channel whether it's okay to deploy.
+2. To start a deployment, push the relevant branch to the `platform` remote.
+3. Update the `#dev-etna` Slack channel when the deployment is complete.
```
Comment (TNA-Allan) [inline .github/PULL_REQUEST_TEMPLATE.md:33]: Deployment mentions in slack is currently given in `#dev-etna-dev` or `respective team channels` for that PR where it requires attention to most of us as we all are in those.

**Review by TNA-Allan — CHANGES_REQUESTED** [review]
@ababic , Nicely categorised :)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (ababic) [reply .github/PULL_REQUEST_TEMPLATE.md:33]: It would be nice if we could make it a bit more consistent. If all devs are in `#dev-etna`, maybe just send there for cross-team visibility without the repetition?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (ababic) [reply .github/PULL_REQUEST_TEMPLATE.md]: I've gone with "Merging a feature or bug fix?", as I don't think it makes sense to be too wordy here.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -17,24 +16,19 @@ Related ticket(s):
...
-        - select `Squash and merge`
-
-- After merging PR:
-
-  - check and arrange to deploy on platform-sh.
-  - update status on JIRA ticket on deployment.
-  - update slack on deployment progress of the JIRA ticket once deployed.
+1. Post to the appropriate Slack channel to check that it is okay to continue.
+2. To start a deployment, push the relevant branch to the `platform` remote.
+3. Update the `#dev-etna` Slack channel when the deployment is complete.
```
Comment (TNA-Allan) [inline .github/PULL_REQUEST_TEMPLATE.md:33]:
```suggestion
3. Update the appropriate Slack channel when the deployment is complete.
```

**Review by TNA-Allan — APPROVED** [review]
Looks good. Thanks @ababic
